ballerina/api_del_doc_with_credit_block/Module.md

```diff
@@ -35,14 +35,14 @@ designed for large-scale enterprises by SAP SE.
 
 ## Quickstart
 
-To use the `sap.s4hana.delivery` connector in your Ballerina application, modify the `.bal` file as follows:
+To use the `sap.s4hana.api_del_doc_with_credit_block` connector in your Ballerina application, modify the `.bal` file as follows:
 
 ### Step 1: Import the module
 
-Import the `sap.s4hana.delivery` module.
+Import the `sap.s4hana.api_del_doc_with_credit_block` module.
 
 ```ballerina
-
+import ballerinax/sap.s4hana.api_del_doc_with_credit_block as deldcwcb;
```

Comment:
```suggestion
import ballerinax/sap.s4hana.api_del_doc_with_credit_block as delDoc;
```

Reply:
[Add PR review changes to gradle build](https://github.com/ballerina-platform/module-ballerinax-sap.s4hana.delivery/pull/7/commits/031c361cb9b165454c9d51d179f5c52002682d85)